tests/type-tests.dx

```diff
@@ -597,10 +597,23 @@ table = [1.0, 2.0]
 > :html table
 >       ^^^^^
 
-
 -- We can't have table literals of non-data
 :p [\x:Float. x, \x:Float. x + x]
 > Type error:Couldn't synthesize a class dictionary for: (Data (Float32 -> Float32))
 >
 > :p [\x:Float. x, \x:Float. x + x]
 >    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+data A = MkA
+
+interface I a
+  m : Unit
+> Error: variable already defined: a
+
+instance I A extraneous given {extraneous}
+  m = todo
+> Error in (earlier) definition of variable: I
```

Comment:
Are these the error messages you meant to test here?  It looks like the error on line 611 wasn't intended, but is an error because `a` is defined above on line 388.